api/swagger.yaml

```diff
@@ -265,6 +265,10 @@ definitions:
               - "rshared"
               - "slave"
               - "rslave"
+          NonRecursive:
+            description: "Disable recursive bind mount."
```

Comment:
We may want to add a bit more information about what this does (also, that it only applies to / can be used for `Type=bind`)

Reply:
This is already in `BindOptions`, I think that's good.